features/buttonComponent.js

```diff
@@ -0,0 +1,37 @@
+/* 
+Component accepts following inputs: label (string), class (string), animate on hover (boolean)
+Component has a light grey background and black text. When hovered, the background changes to dark grey and the label is white.
+If animate on hover is true, the button will show an icon of Koala to the right of it
+*/
+import './buttonComponentStyles.css';
+
+export default function createButton(label, classStyle, animate) {
+  const koalaButton = document.createElement('div');
+  koalaButton.classList.add('koala-button');
```

Comment:
Try to use this part only if animate === true - we don;t want every button to be wrapped in koala stuff